docs/software/apps-and-envs/orca.md

```diff
@@ -0,0 +1,62 @@
+# ORCA
+
+[ORCA](https://www.faccts.de/orca/) is a general-purpose quantum chemistry package that supports a wide variety of methods such as density functional theory, many-body perturbation and coupled cluster. ORCA is free of charge, but not open source.
+
+Check out the ORCA manual for more information: https://www.faccts.de/docs/orca/6.0/manual/
+
+Keywords: `quantum chemistry`, `chemistry`
+
+## Available modules
+There are several ORCA modules on Midway3 that you can check via `module avail orca`:
+
+===+ "Midway3"
+    ```
+    ---------------------------- /software/modulefiles -----------------------------
+    orca/4.1  orca/4.2.1  orca/5.0.2(default)  orca/6.0.1  
+    orca/4.2  orca/5.0.0  orca/5.0.4
+    ```
+
+You can use `module show` to see the dependencies and environment variables set by a specified `orca` module, for example, `module show orca/6.0.1`:
+
+```
+-------------------------------------------------------------------
+/software/modulefiles/orca/6.0.1:
+
+module-whatis   {setup orca 6.0.1 compiled with the system compiler}
+conflict        orca
+module          load openmpi/4.1.2+gcc-10.2.0
+prepend-path    PATH /software/orca-6.0.1-el8-x86_64/bin
+setenv          ORCA_DIR /software/orca-6.0.1-el8-x86_64/bin
+-------------------------------------------------------------------
+```
+
+
+## Example job script
+
+An example batch script to run ORCA for Midway3 is given as below
```

Comment:
"An example batch script to run ORCA for Midway3 is given as below" => "An example batch script to run ORCA on Midway3 is given below"?

- Similarly for all other .md's examples as well?